src/utils/dbHelper.js

```diff
@@ -26,11 +26,27 @@ class Database {
    * @returns {void}
    */
   _connect() {
-    mongoose.connect(this.uri, this.options).then(() => {
-      // eslint-disable-next-line no-console
-      console.log("Database connection successful");
-    });
+    mongoose.connect(this.uri, this.options);
   }
 }
 
-module.exports = new Database();
+/**
+ * Closes connection from MongoDB server
+ * @returns {Promise} Empty promise
+ */
+const closeConnection = () => {
```

Comment:
Instead of creating a separate function, you can add a method to the `Database` class itself. It'd make more sense.